tests/Makefile

```diff
@@ -49,7 +49,7 @@ endif
 ifdef WINDOWS_BUILD
 DLEXT=dll
 EXEXT=.exe
-LOCAL_LDFLAGS += -lws2_32
+LOCAL_LDFLAGS += -lws2_32 -lbcrypt
```

Comment:
Why are we not adding this for cmake?

Reply:
Because in cmake it is set once and included by the targets.